src/district0x/ui/history.cljs

```diff
@@ -1,10 +1,30 @@
 (ns district0x.ui.history
-  (:require [district0x.ui.utils :as d0x-ui-utils]
+  (:require [bidi.bidi :as bidi]
+            [clojure.string :as string]
+            [district0x.ui.utils :as d0x-ui-utils]
+            [medley.core :as medley]
             [pushy.core :as pushy]
             [re-frame.core :as re-frame]))
 
+(goog-define pushroute-hosts "0.0.0.0")
+
 (def routes (atom nil))
 
+(defn hashroutes? []  
```

Comment:
You can actually make this as a `(def hashroutes? ...`, not as a function. Will help the performance